### PR TITLE
Dev/fix dnd

### PR DESCRIPTION
**Pull request by my8bird**

### CODE REVIEW

File: src/app/board/columns/issue_columns.tpl.html [hunk 1]
```diff
@@ -15,17 +15,17 @@
...
-      <a class="btn btn-success"
-         ng-href="https://github.com/{{repoModel.owner}}/{{repoModel.repo}}/issues/new"
-         target="_blank">Create Issue</a>
-
       <!-- CARDS List -->
-      <ul class="column-body" ui-sortable="colCtrl.geSortableOptions()" >
-        <div class="card_wrapper"
-             ng-repeat="issue in repoModel.issues | issuesInBacklog | filter:repoModel.cardSearchText"
+      <ul class="column-body" ng-model="colCtrl.issues"
+          ui-sortable="colCtrl.getSortableOptions()" >
```
Comment (my8bird) [inline src/app/board/columns/issue_columns.tpl.html:20]: The fix for why the backlog did not work is that `geSortableOptions` does not equal `getSortableIOptions`

File: src/common/github_api/github_api.js [hunk 2]
```diff
@@ -20,7 +20,8 @@ angular.module('github.api', ['restangular'])
  @description
  Angular service `gh` which provides tools for accessing GitHub API's
  */
-.service('gh', function gh(GitHubRestangular, $http, $interpolate, $rootScope, $q) {
+
+.service('gh', function gh($q, GitHubRestangular) {
```
Comment (my8bird) [inline src/common/github_api/github_api.js:24]: i just cleaned up the deps for this service

File: src/app/services/repos_service.js [hunk 3]
```diff
@@ -61,11 +61,7 @@ angular.module('Trestle')
    this._loadIssues = function() {
       gh.listRepoIssues(trRepoModel.owner, trRepoModel.repo)
          .then(function(issues) {
-            // Pre sort the list of issues so that the jquery sortable plugin
-            // works correctly.
-            trRepoModel.issues = _.sortBy(issues, function(issue) {
-               return issue.config.weight;
-            }).reverse();
+            trRepoModel.issues = issues;
```
Comment (my8bird) [inline src/app/services/repos_service.js:64]: the sorting takes place in a different place

File: src/app/board/columns/issue_column_ctrl.js [hunk 4]
```diff
@@ -13,17 +13,36 @@ angular.module('Trestle.board')
       this.columnName = (this.isBacklog ? 'Backlog' : this.labelName);
 
       $scope.$id = "ColumnCtrl_" + this.columnName + $scope.$id;
+
+      this.issues = [];
+      this._updateIssueList();
+      trRepoModel.$watch('issues', angular.bind(this, this._updateIssueList), true);
```
Comment (my8bird) [inline src/app/board/columns/issue_column_ctrl.js:19]: In order for the sortable plugin to work we need an actual list that can be mutated by the plugin.

**Comment by abierbaum** [comment]
+1

**Pull request merged**